.gitignore

```diff
@@ -7,4 +7,13 @@ project.properties
 .idea/
 .project
 .settings/
-.classpath
\ No newline at end of file
+.classpath
+runelite-client/src/main/java/net/runelite/client/plugins/.DS_Store
```

Comment:
This shouldn't be here, maybe it can be submitted as a separate PR but it seems to be here on accident